zuul.sf.d/nodesets.yaml

```diff
@@ -66,6 +66,18 @@
         nodes:
           - eos-4.24.6
 
+- nodeset:
+    name: eos-4.24.6-python39
```

Comment:
Just keep one nodeset please. BTW, both Fedora-35 and CentOS8 come with the Python versions we need.